src/main/java/com/rtb/projectmanagementtool/task/TaskServlet.java

```diff
@@ -41,7 +41,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     // Get Task
     long taskID = Long.parseLong(request.getParameter("taskID"));
     TaskController taskController = new TaskController(datastore);
-    // TaskData task = taskController.getTaskByID(taskID);
+    if (taskID != 0 && taskID != 1) {
+      task1 = taskController.getTaskByID(taskID);
+    }
     // ArrayList is for HashMap below. Is there a better way to do this?
     ArrayList<TaskData> taskInArrayList = new ArrayList<>(Arrays.asList(task1));
```

Comment:
Is it necessary to put this ArrayList in a HashMap? I believe you could just send this ArrayList directly to the response writer in line 76

Reply:
I can, but I'm putting it into a HashMap because later I'm going to uncomment most of the code below (lines 50-65) and put all of that in the HashMap as well (lines 72-75) and send that to the response writer all together. Then everything will be clearly accessible on the JavaScript end (lines 19-23 in task.js).